ui/components/DashboardComponent.js

```diff
@@ -1,597 +1,658 @@
-            onClick={() => this.handleConfigure("prometheus")}
-          >
-            <SettingsIcon className={classes.settingsIcon} />
-            Configure Prometheus
-          </Button>
-        </div>
-      );
-    }
-    if (prometheus && prometheus.prometheusURL && prometheus.prometheusURL !== "") {
-      showPrometheus = (
-        <Chip
-          label={prometheus.prometheusURL}
-          onClick={self.handlePrometheusClick}
-          icon={<img src="/static/img/prometheus_logo_orange_circle.svg" className={classes.icon} />}
-          className={classes.chip}
-          key="prom-key"
-          variant="outlined"
-        />
-      );
-    }
-
-    const showMetrics = (
-      <div>
-        {showPrometheus}
-        {showGrafana}
-      </div>
-    );
-
-    const showRelease = (
-      //Add Release Version Text
-      <FormControl component="fieldset">
-        <FormLabel component="legend" style={{ fontWeight: "bold" }}>
-          Release Channel
-        </FormLabel>
-        <RadioGroup aria-label="release_channel_option" name="release_channel">
-          <FormControlLabel value="stable_channel" disabled control={<Radio checked={true} />} label="Stable Channel" />
-          <FormControlLabel value="edge_channel" disabled control={<Radio />} label="Edge Channel" />
-        </RadioGroup>
-      </FormControl>
-    );
-
-    return (
-      <NoSsr>
-        <div className={classes.root}>
-          <Typography variant="h6" gutterBottom className={classes.chartTitle}>
-            Connection Status
-          </Typography>
-
-          <Grid container spacing={1}>
-            <Grid item xs={12} md={6}>
-              {self.showCard("Kubernetes", showConfigured)}
-            </Grid>
-            <Grid item xs={12} md={6}>
-              {self.showCard("Adapters", showAdapters)}
-            </Grid>
-            <Grid item xs={12} md={6}>
-              {self.showCard("Metrics", showMetrics)}
-            </Grid>
-            <Grid item xs={12} md={6}>
-              {self.showCard("Release", showRelease)}
-            </Grid>
-          </Grid>
-        </div>
-      </NoSsr>
-    );
-  };
-
-  render() {
-    return this.configureTemplate();
-  }
-}
-
-DashboardComponent.propTypes = {
-  classes: PropTypes.object.isRequired,
-};
-
-const mapDispatchToProps = (dispatch) => ({
-  updateProgress: bindActionCreators(updateProgress, dispatch),
-});
-const mapStateToProps = (state) => {
-  const k8sconfig = state.get("k8sConfig").toJS();
-  const meshAdapters = state.get("meshAdapters").toJS();
-  const meshAdaptersts = state.get("meshAdaptersts");
-  const grafana = state.get("grafana").toJS();
-  const prometheus = state.get("prometheus").toJS();
-  return {
-    meshAdapters,
-    meshAdaptersts,
-    k8sconfig,
-    grafana,
-    prometheus,
-  };
-};
-
-export default withStyles(styles)(
-  connect(mapStateToProps, mapDispatchToProps)(withRouter(withSnackbar(DashboardComponent)))
-);
+import React from "react";

```

Comment:
Can you fix this, why is there change at everyline?

Reply:
Actually I couldn't understand this behavior either as when I see diff in my system it correctly points out the changes while here it doesn't. I have tried running `eslint --fix` as well as have editorconfig plugin installed. Can you please suggest a solution for the same?

Reply:
Try correcting your editor configuration, this has been done by your editor/IDE. This makes it very difficult to review as we don't know which lines have been added or delete, or amended.

Reply:
I have fixed the diff issue. The problem was apparently in .editorconfig's `end_of_line: crlf`. I disabled editorconfig plugin on my text editor(vscodium) and that kind of fixed the issue for me. I assume it could be due to the way windows handles end of lines (CRLF) and the way linux handles it (LF).